test/java/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelUnitTest.java

```diff
@@ -65,31 +66,37 @@ public void it_initializes_UI_fields_with_values_from_the_initial_configuration(
   }
 
   @Test
-  public void shows_an_error_and_resets_the_field_when_the_UI_tries_to_set_a_date_range_end_that_is_before_its_start() {
+  public void cannot_insert_an_invalid_end_date() {
```

Comment:
I think these tests are safe to remove because the UI tests on `ConfigurationPanelTest` are easier to understand and they're testing the same things...

Reply:
Ok